OTGroundTruther/model/event.py

```diff
@@ -216,7 +218,22 @@ def _exceeds_maximum_events(self, events: list[dict]) -> bool:
         return False
 
     def _get_video_file_name(self, event: dict) -> str:
-        return event[VIDEO_NAME]
+        video_name = event[VIDEO_NAME]
+        if Path(video_name).suffix == "":
+            return video_name + DEFAULT_VIDEO_FILE_SUFFIX
+        elif self._is_detection_file(video_name=video_name):
+            return Path(video_name).stem + DEFAULT_VIDEO_FILE_SUFFIX
+        else:
+            return video_name
+
+    def _is_detection_file(self, video_name: str) -> bool:
+        if (
+            video_name[-1 * len(DEFAULT_DETECTION_FILE_SUFFIX) :]
```

Comment:
Looks complicated, can´t you get the current suffix with `Path(video_name).suffix` and compare this to `DEFAULT_DETECTION_FILE_SUFFIX`?

Reply:
Dont know why i made it so complicated, lol